claimstore/modules/claims/models.py

```diff
@@ -42,9 +44,8 @@ class Claim(db.Model):
         default=lambda: str(uuid4())
     )
     created = db.Column(
-        db.DateTime,
-        nullable=False,
-        default=db.func.current_timestamp()
+        UTCDateTime,
+        nullable=False
     )
```

Comment:
Nobody is really using ClaimStore much yet, so this comment is for the future, just to say to think about future Alembic based DB structure upgrade recipes.